.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint, Test, Build
+
+on:
+  push:
+    tags-ignore:
+    - 'v*'
+    branches:
+    - "**"
+  pull_request:
+
+jobs:
+  lintTestBuild:
+    name: Lint, Test, Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15.6'
```

Comment:
1.16.

Reply:
We're actually going to wait on switching the `test` job to go1.16 because there broken tests and linter warning that we'd like to fix first.